src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -128,7 +128,7 @@ public long GetMinimalEtagForReplication()
                     foreach (var external in externals)
                     {
                         var state = GetExternalReplicationState(_server, Database.Name, external.TaskId, ctx);
-                        var myEtag = ChangeVectorUtils.GetEtagById(state.DestinationChangeVector, Database.DbBase64Id);
+                        var myEtag = ChangeVectorUtils.GetEtagById(state.SourceChangeVector, Database.DbBase64Id);
```

Comment:
this needs to be changed in 5.x ?

Reply:
Not sure, will see the results of the tests suit :)